frontend/app/views/notes/_form.html.erb

```diff
@@ -1,18 +1,26 @@
-<% 
+<%
+  # define these when nesting notes within a subrecord
+  nested_note_jsonmodel ||= nil
+  nested_in_jsonmodel ||=nil
+
+  section_id = nested_note_jsonmodel unless nested_note_jsonmodel.nil?
   section_id = "notes" if section_id.blank?
-  all_note_types ||= nil 
+  header_size = "h3" if header_size.blank?
+  all_note_types ||= nil
 %>
-<%= render_aspace_partial :partial => "notes/template", :locals =>  { :all_note_types => all_note_types, :form_note_type => section_id  }  %>
+<%= render_aspace_partial :partial => "notes/template", :locals =>  { :all_note_types => all_note_types, :form_note_type => section_id, :nested_in_jsonmodel => nested_in_jsonmodel } %>
 
-<section id="<%= section_id %>" class="subrecord-form notes-form  <%= "note-inline" if inline? %>">
-  <h3 class="subrecord-form-heading">
+<section id="<%= section_id %>"
+         class="subrecord-form notes-form  <%= "note-inline" if inline? %>"
+         <%= "data-template=template_#{nested_note_jsonmodel}_note_type_selector_inline" if nested_note_jsonmodel %>>
```

Comment:
We need to let the javascript now which template we're dealing with.  A side effect of having notes nested.